Editor/PackageDependencies/XRInteractionPackageEnabler.cs

```diff
@@ -1,17 +1,10 @@
-﻿using System;
-using UnityEditor;
-using System.Linq;
-using System.Collections.Generic;
-
-namespace Innoactive.CreatorEditor.PackageManager.XRInteraction
+﻿namespace Innoactive.CreatorEditor.PackageManager.XRInteraction
 {
     /// <summary>
     /// Adds Unity's XR-Interaction-Toolkit package as a dependency and sets specified symbol for script compilation.
     /// </summary>
-    public class XRInteractionPackageEnabler : Dependency, IDisposable 
-    {
-        private const string CreatorXRInteractionSymbol = "CREATOR_XR_INTERACTION";
-        
+    public class XRInteractionPackageEnabler : Dependency 
```

Comment:
I do not understand how does it work but I absolutely love it.

Reply:
Lovely magic!